scrapper.py

```diff
@@ -44,20 +77,120 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(self.article_url, self.article_id)
+
+    def parse(self):
+        response = requests.get(self.article_url, headers=HEADERS)
+        time.sleep(random.randint(1, 3))
```

Comment:
this should be implemented in a logic outside the parse method - somewhere where you call it